sdk/communication/AzureCommunicationChat/Tests/ChatThreadClientUnitTests.swift

```diff
@@ -45,8 +44,8 @@ class ChatThreadClientUnitTests: XCTestCase {
     private let settings = TestSettings.loadFromPlist()
 
     override func setUpWithError() throws {
-        let endpoint = settings.endpoint ?? "https://endpoint"
-        let token = settings.token ?? generateFakeToken()
+        let endpoint = settings?.endpoint ?? "https://endpoint"
+        let token = settings?.token ?? generateFakeToken()
```

Comment:
Is this token and endpoint part not related to test recording? I notice that it's still in the file but AzureTest had its import removed.

Reply:
They are used to construct the client. They aren't needed by AzureTest, per se

Reply:
Ok I was just wondering :)